### DISCUSSION TITLE
Add chat to VC room

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#152] [Feature Suggestions]
The VC room currently has a large blank area, where chat could possibly go! I've always wanted dedicated chat rooms for each VC in Discord, and it seems like there's already a large space where it could be placed in revolt.

**Comment by n1ckoates** [comment]
This is quite an obvious suggestion, but VC chat shouldn't be stored, and you should only be able to view messages sent *after* you joined the VC.
